src/lib/sheet.ts

```diff
@@ -201,5 +207,6 @@ const setMatches = (match_array: number[][]) => {
   matches.set(match_obj);
 };
 
+//Im not really sure what this does but it is necessary for setform
```

Comment:
This is a matrix transpose the data is in a 2d array but the columns and rows are incorrect for what we want to do because Google sheets gives us an array of columns but we want an array of rows I think that's what it is atleast haven't touched this code in a while